app/src/main/res/menu/menu_cancel_manager.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    >
+    <item
+        android:id="@+id/delete"
+        android:icon="@null"
+        android:title="@string/delete"
+        app:showAsAction="never"/>
+    <item
+        android:id="@+id/cancel_manager"
+        android:icon="@null"
+        android:title="Cancel Manager"
```

Comment:
WARNING: [HardcodedText] [I18N] Hardcoded string "Cancel Manager", should use `@string` resource
 Hardcoding text attributes directly in layout files is bad for several reasons:

* When creating configuration variations (for example for landscape or portrait)you have to repeat the actual text (and keep it up to date when making changes)

* The application cannot be translated to other languages by just adding new translations for existing string resources.

There are quickfixes to automatically extract this hardcoded string into a resource lookup.